api/sample.env

```diff
@@ -167,6 +167,22 @@ TEST_DATABASE_URL=postgresql://postgres@localhost/pix_test
 # default: none
 # SENDINBLUE_PASSWORD_RESET_TEMPLATE_ID=
 
+# String to allow links in emails to redirect to a specific domain when user comes
+# from french domain
```

Comment:
String for links in emails redirect to a specific domain when user comes from french domain